AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.txt

```diff
@@ -154,10 +165,10 @@ private var runtimeTargets: [Target] {
                 .smithyChecksums,
                 .smithyWaitersAPI,
                 .awsSDKCommon,
-                .awsSDKIdentity,
                 .awsSDKHTTPAuth,
                 .awsSDKEventStreamsAuth,
                 .awsSDKChecksums,
+                "InternalAWSCommon",
```

Comment:
In this & the other 2 internal clients, the AWSSDKIdentity dependency is removed since that module is no longer referenced in the client.  The InternalAWSCommon module is added since that is where the internal clients get their default identity resolvers from.